client/src/components/chat/Chat.js

```diff
@@ -178,20 +185,6 @@ class Chat extends Component {
           </Header>
         }
 
-        { type === 'APPROVER' && selectedProposal && title && subtitle &&
```

Comment:
These blocks are all new and should not have been removed. This will result in chat being completely broken. Make sure you test the application before and after in the browser in the manner I demoed early today to check for regressions.